packages/messaging/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/react-native-aepmessaging",
-  "version": "1.0.0-alpha.2",
+  "version": "1.0.0-beta.1",
```

Comment:
@yangyansong-adbe plans to update all the versions to beta once in staging, so we can revert this for now